src/mSupplyMobileApp.js

```diff
@@ -83,21 +83,27 @@ class MSupplyMobileAppContainer extends React.Component {
     this.handleBackEvent = this.handleBackEvent.bind(this);
     this.getCanNavigateBack = this.getCanNavigateBack.bind(this);
     this.renderPageTitle = this.renderPageTitle.bind(this);
-    this.scheduler.schedule(this.synchronise,
-                            SYNC_INTERVAL);
+    this.scheduler.schedule(this.synchronise, SYNC_INTERVAL);
     this.scheduler.schedule(() => {
-      if (this.state.currentUser !== null) { // Only reauthenticate if currently logged in
+      if (this.state.currentUser !== null) {
+        // Only reauthenticate if currently logged in
         this.userAuthenticator.reauthenticate(this.onAuthentication);
       }
     }, AUTHENTICATION_INTERVAL);
   }
 
   componentDidMount() {
     BackHandler.addEventListener('hardwareBackPress', this.handleBackEvent);
+    if (this.state.isInitialised && this.synchroniser.isSyncing()) {
```

Comment:
That's a good question @andreievg. If you're right, we need to change it in the page specific listeners too